src/components/3d_editor/advanced_geometry/python_transformation/PythonTransformation.tsx

```diff
@@ -122,8 +123,8 @@ class PythonTransformation extends React.Component<
                 title="Python Transformation"
                 isSubmitButtonDisabled={isLoading || isRunning}
             >
-                <DialogContent>
-                    <PyodideLoader onLoad={this.onPyodideLoad} triggerLoad={show} />
+                <PyodideLoader onLoad={this.onPyodideLoad} triggerLoad={show} />
+                <Box mt={theme.spacing(1)} p={`0 ${theme.spacing(3)}`}>
```

Comment:
In order to make RUN button reachable even if we scroll the code display.  
![image](https://github.com/Exabyte-io/materials-designer/assets/79542055/93a530a6-0301-4820-b00e-3f25abee5b82)
Other solution is to define custom Header, with Ctrl+C, Ctrl+V title, onClose button, margins, and add this indise of it.